ci/tcf-tests.yaml

```diff
@@ -25,7 +25,7 @@ services:
         - no_proxy
     working_dir: "/project/TrustedComputeFramework/"
     entrypoint: "bash -c \"\
-        source scripts/tcs_startup.sh -y \
+        source scripts/tcs_startup.sh -y -s -l http://avalon-lmdb:9090 \
```

Comment:
Why don't you remove running kv_storage service from tcs_startup.sh at all. Now, this is a seperate service, be it standalone or docker mode, we should start this service seperately.

Reply:
@Ram-srini: consider both PR #214 and #217. PR #217 builds on the changes in PR #214 These PRs are urgently needed to fix the regressions in Avalon so I can deliver a working version of Avalon that is needed by a partner who wants to use Avalon.

@pankajgoyal2 `tcs_startup.sh` starts all the services needed by Avalon in one simple startup script, including the enclave manager and listener. Avalon is very complex and there is no need to make it more difficult to use.

Reply:
@Ram-srini: to reduce confusion I removed the duplicate changes from PR #214.  Please consider both PR #214 and this PR separately.  I will merge after approval.